lighthouse-core/gather/gatherers/seo/robots-txt.js

```diff
@@ -48,8 +48,8 @@ class RobotsTxt extends FRGatherer {
 
     const robotsUrl = new URL('/robots.txt', passContext.url).href;
     await passContext.driver.fetcher.enable();
-    return passContext.driver.fetcher.fetchResource(robotsUrl)
-      .catch(() => ({status: null, content: null}));
+    return passContext.driver.fetcher.fetchResource(robotsUrl, {timeout: 1_000})
```

Comment:
shouldn't we tweak the method's default instead? and since mostly all usages of this are done in parallel (source maps), I'd be comfortable upping it to 2s.

Reply:
sure, wfm